include/scrcmd_message.h

```diff
@@ -0,0 +1,31 @@
+#ifndef POKEHEARTGOLD_SCRCMD_MESSAGE_H
+#define POKEHEARTGOLD_SCRCMD_MESSAGE_H
+
+#include "field_system.h"
+#include "message_format.h"
+#include "msgdata.h"
+#include "script.h"
+
+typedef struct MessageBox {
+    STRING *message;
+    STRING *buffer;
+    MessageFormat *messageFormat;
+    WINDOW *window;
+    u8 *unk10;
+    u8 *textPrinterNumPtr;
+} MessageBox;
+
+BOOL ov01_021EF348(SCRIPTCONTEXT *ctx);
```

Comment:
if these aren't referenced in any other file, they should be at the top of the .c and marked as static (sorry for any confusion that may have come from this)